RubberduckBaseProject.csproj

```diff
@@ -63,7 +63,8 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(Configuration)' == 'Release'">
-    <DebugType>pdbonly</DebugType>
+    <DebugType>none</DebugType>
+    <DebugSymbols>false</DebugSymbols>
```

Comment:
This should be the only necessary introduction of the `DebugType` and `DebugSymbols` properties. All other (relevant) projects import this project and shouldn't override the values here.

Could you try reverting the changes to the other `.csproj` files and check whether it works as expected?